### ISSUE TITLE
Build fails for CMake v3.5

### ISSUE THREAD

**Issue by MaximeVdB** [dftbplus/dftbplus#649]
**Describe the bug**
The `INSTALL.rst` file says that CMake >= v3.5 is required, but my build of the current master fails with CMake v3.5:
```bash
CMake Error at CMakeLists.txt:19 (GNUInstallDirs_get_absolute_install_dir):
  Unknown CMake command "GNUInstallDirs_get_absolute_install_dir".
```
The v20.1 release does get built with CMake v3.5 without problems.

It looks like commit c0cfb93 introduced the `GNUInstallDirs_get_absolute_install_dir` function which only got introduced in CMake v3.7 (see https://cmake.org/cmake/help/v3.13/release/3.7.html#modules).

**Comment by awvwgk** [comment]
We are planning to update the CMake requirement to 3.16 for the next release, which should kinda fix this issue, see #636.

**Comment by aradi** [comment]
@MaximeVdB The respective changes are on the main branch now. If you experience any issues, let me know.
